### PR TITLE
Visualization demo notebook

### PR DESCRIPTION
**Pull request by rezaarezvan**
First of all, super interesting project and overall a super nice codebase to work with :) Good code quality and the documentation is relevant and helpful, thanks!

I started out with trying out a pretrained checkpoint and trying the `predict` and `evaluate` scripts, worked great, but I wanted a bit more of a visualization and demo/example to get started, so I made one!

I might have misused some functions but when writing this I had to fix two things.

Firstly,

`"mask": batch.mask.long().to(DEVICE)  # Ensure mask is long type, for some reason it breaks if it isn't set to `long``

and Secondly,

`times = torch.tensor([t], device=DEVICE, dtype=torch.float32) # Same here, for some reason I have to ensure float32, otherwise it breaks`

I think it is easier understood viewing the notebook where these are but I provide them here as well.

### CODE REVIEW

**Comment by rssrwn** [comment]
Thanks for the interest in the project! This visualisation notebook seems very useful. I had something similar on a separate repo but just never committed it.

Yeah, 32 bits for floats and 64 bits for masks is generally what is used throughout, so it probably just needs to be kept consistent.

**Review by rssrwn — APPROVED** [review]
